cypress/e2e/tests/migration/applicationinventory/assessment/assess_review.test.ts

```diff
@@ -165,8 +176,10 @@ describe(["@tier1"], "Application assessment and review tests", () => {
         application.verifyStatus("assessment", "Completed");
         application.verifyStatus("review", "Completed");
 
-        // todo: Automate the bug once it's fixed
-        // bug: https://issues.redhat.com/browse/MTA-1751
+        // Automates bug: https://issues.redhat.com/browse/MTA
```

Comment:
where isthe bug number ?

Reply:
opened a PR for this: https://github.com/konveyor/tackle-ui-tests/pull/858